gradle/libs.versions.toml

```diff
@@ -0,0 +1,40 @@
+[versions]
+kotlin-version = "1.9.21"
+
+# libs
+crt-java-version = "0.29.1"
+coroutines-version = "1.7.3"
+
+# testing
+junit-version = "5.10.1"
+kotest-version = "5.8.0"
+mock-server-version = "5.15.0"
+# fixme - remove in favor of clikt
```

Comment:
Should we add a task in the backlog in addition to this comment?

Reply:
it's not a public dependency, only consumed in the elasticurl example/test application. This was more a reminder that it was deprecated and is basically unmaintained at this point